### ISSUE TITLE
Expected status while sync is in progress?

### ISSUE THREAD

**Issue by LukeOwlclaw** [nextcloud/desktop#3298]
### Expected behaviour
When sync is in progress of any repository, ie. when files are being transferred, the tray icon should show the blue symbol. (Note the subtle difference to https://github.com/nextcloud/desktop/issues/619 which is about "checking for changes".)

### Actual behaviour
Sometimes it is green, sometimes blue, sometimes yellow.

### Steps to reproduce
Unfortunately I was not able to reproduce the issue. It just happens sometimes. But it happened to me on at least three machines over the last three years.

### Client configuration
Client version: 3.2.1
Operating system: Win 10

### Server configuration
Nextcloud version: 20.0.9.1

### Questions

I am really confused about this issue because of multiple reasons:
- There is issue https://github.com/nextcloud/desktop/issues/619 which is about about the tray icon being green while checking for changes (but not while the actual sync is in progress)
- There is issue https://github.com/nextcloud/desktop/issues/2210 which is discussing which states to show in the systray but it not even mentions the "blue state".
- I see all kinds of weird state combination now and then: E.g.
![image](https://user-images.githubusercontent.com/2737900/117635916-99407280-b180-11eb-980d-d41bff9e356f.png)
(but actually files were syncing, so definitely not offline)
![image](https://user-images.githubusercontent.com/2737900/117636039-b2492380-b180-11eb-9787-e93e26d2cfaf.png)
(the other repository was also in blue state (syncing; not yellow))
![image](https://user-images.githubusercontent.com/2737900/117636127-cb51d480-b180-11eb-9e5e-53ec1157b381.png)
- I am having this problem for a very long time (note that #619 which is very similar/related is more than 2.5 years old) but there is not even an issue which addresses this exact problem. Am I the only person having this problem?
- Uncountable times I wondered why my files aren't available because the green tray icon showed that everything is in sync but it wasn't. As put by https://github.com/nextcloud/desktop/issues/619#issuecomment-708621154 this is the second worst bug a sync client can have. And for me (and it seems **only** me) it is happening for years. I am doing something wrong?

**Comment by LukeOwlclaw** [comment]
Okay then, it is really just me. 🤔

**Comment by mgallien** [comment]
> Okay then, it is really just me. thinking

Sorry for the delay
We added a status feature that is now synced from server to desktop. This is what you see in the main dialog in from of your avatar. This is not the state of the sync.
It would be nice if you update to the latest release and test if you can reproduce problems with out of sync status icon and post the logs here. That would allow us to fix that problem and help you.

**Comment by LukeOwlclaw** [comment]
I updated to 3.2.4 and I got a new weird combination of status states:

![image](https://user-images.githubusercontent.com/2737900/126661193-51f85037-0fcf-4a93-91d9-0d79e457fd56.png)

This happened after the server was for a longer time in maintenance mode. Now it is back up, as correctly shown by the green symbol and the "online" string.  It even synced a file 8 minutes ago. But the tray bar icon is still gray.
